routes/gamesRouter.js

```diff
@@ -1,163 +1,176 @@
-    const ageRating = req.body.ageRating;
-    const violence = req.body.violence;
-    const sexAndNudity = req.body.sexAndNudity;
-    const alcoholAndDrugs = req.body.alcoholAndDrugs;
-    const gambling = req.body.gambling;
-    const explicitLanguage = req.body.explicitLanguage;
-    const averageScore = req.body.averageScore;
-    const publisher = req.body.publisher;
-    const releaseDate = req.body.releaseDate;
-    new Game({ title, 
-        category, 
-        ageRating, 
-        violence, 
-        sexAndNudity, 
-        alcoholAndDrugs, 
-        gambling, 
-        explicitLanguage,
-        averageScore,
-        publisher,
-        releaseDate}).save((err, result) => {
-        res.status(200).send({ status: result });
-    });
+router.post("/add", (req, res) => {
+  const {
+    title,
+    category,
+    ageRating,
+    violence,
+    sexAndNudity,
+    alcoholAndDrugs,
+    gambling,
+    explicitLanguage,
+    averageScore,
+    publisher,
+    releaseDate,
+  } = req.body;
+  new Game({
+    title,
+    category,
+    ageRating,
+    violence,
+    sexAndNudity,
+    alcoholAndDrugs,
+    gambling,
+    explicitLanguage,
+    averageScore,
+    publisher,
+    releaseDate,
+  }).save((err, result) => {
+    res.status(200).send({ status: result });
+  });
 });
 
-router.post('/search', (req, res) => {
-    const title = req.body.title;
-    const category = req.body.category;
-    const ageRating = req.body.ageRating;
-    const violence = req.body.violence;
-    const sexAndNudity = req.body.sexAndNudity;
-    const alcoholAndDrugs = req.body.alcoholAndDrugs;
-    const gambling = req.body.gambling;
-    const explicitLanguage = req.body.explicitLanguage;
-    const averageScore = req.body.averageScore;
-    const publisher = req.body.publisher;
-    const releaseDate = req.body.releaseDate;
-    const query = {};
-    if (title) {
-        query.title = title;
-    }
-    if (category) {
-        query.category = category;
-    }
-    if (ageRating) {
-        query.ageRating = ageRating;
-    }
-    if (violence) {
-        query.violence = violence;
-    }
-    if (sexAndNudity) {
-        query.sexAndNudity = sexAndNudity;
-    }
-    if (alcoholAndDrugs) {
-        query.alcoholAndDrugs = alcoholAndDrugs;
-    }
-    if (gambling) {
-        query.gambling = gambling;
-    }
-    if (explicitLanguage) {
-        query.explicitLanguage = explicitLanguage;
-    }
-    if (averageScore) {
-        query.averageScore = averageScore;
-    }
-    if (publisher) {
-        query.publisher = publisher;
-    }
-    if (releaseDate) {
-        query.releaseDate = releaseDate;
-    }
+router.post("/search", (req, res) => {
+  const {
+    title,
+    category,
+    ageRating,
+    violence,
+    sexAndNudity,
+    alcoholAndDrugs,
+    gambling,
+    explicitLanguage,
+    averageScore,
+    publisher,
+    releaseDate,
+  } = req.body;
+  const query = {};
+  if (title) {
+    query.title = title;
```

Comment:
You could save the variable unpacking and just use `query.title = req.body.title`

Reply:
Good point.